src/web/lib/components/PaginatedThemeSelector/index.js

```diff
@@ -27,71 +28,81 @@ export const PaginatedThemeSelector = ({
   const nextDisabled = currentPage + 1 >= pageCount;
 
   return (
-    <div className={classNames("theme-selector", className)}>
-      {page.map(([key, { theme }]) => {
-        const customImages = (theme.images.custom_backgrounds || []).map(
-          item => {
-            const customImage = { ...item };
-            customImage.image = images[item.name].image;
-            return customImage;
-          }
-        );
-        return (
-          <div
-            key={key}
-            className={classNames("theme-selector-preview", previewClassName)}
-          >
-            {onDelete && (
-              <button
-                className="delete-theme"
-                onClick={() => onDelete(key)}
-                title="Delete"
-              >
-                <img src={iconClose} />
-              </button>
-            )}
-            <Browser
-              {...{
-                size: "small",
-                theme,
-                customImages,
-                onClick: () => onClick(theme)
-              }}
-            />
-          </div>
-        );
-      })}
-      {pageCount > 1 && (
-        <footer className="page-selector">
-          <button
-            className={classNames("previous", { disabled: prevDisabled })}
-            disabled={prevDisabled}
-            onClick={
-              prevDisabled ? () => {} : () => setCurrentPage(currentPage - 1)
+    <>
+      <div className={classNames("theme-selector", className)}>
+        {page.map(([key, { theme }]) => {
+          const customImages = (theme.images.custom_backgrounds || []).map(
```

Comment:
This looks almost the same thing done in `src/web/lib/components/App/index.js` at line 103, but in that version we are returning an empty object if we are unable to find the custom image, while here we are returning `{ ...item }` (as in the previous version of the code in `src/web/lib/components/App/index.js`).

it there a reason for the change applied to the function defined in `src/web/lib/components/App/index.js` that is not needed here?

Reply:
yes, you're right. good catch. The App file didn't have this extra check here (on line 37) so it didn't cause errors like the one in App. So yes, I'll either refactor this into a function or I'll update the logic in the App - so it's also returning an object with the undefined image property instead of the empty object. It seems either is okay but I'll keep it consistent with this one.

thanks:)